sigs.yaml

```diff
@@ -53,18 +53,6 @@ sigs:
           owners:
             - https://raw.githubusercontent.com/kubevirt/kubevirtci/main/OWNERS
             - https://raw.githubusercontent.com/kubevirt/project-infra/main/OWNERS
-    - dir: sig-virtctl
```

Comment:
Reminder: the SIGs defined in this file are also used as base for the repository groups inside `devstats` ([example](https://kubevirt.devstats.cncf.io/d/1/activity-repository-groups?orgId=1), take a look at the Repository Group selector, containing `SIG virtctl`

So, if we are removing this sig, we should move the OWNERS pointer to maybe the `sig-compute`.

Reply:
Ack, thanks!